server/datastore/mysql/scripts.go

```diff
@@ -2059,16 +2060,16 @@ WHERE
 		return nil, ctxerr.Wrap(ctx, err, "selecting execution information for bulk execution summary")
 	}
 
-	summary.NumTargeted = temp_summary.NumTargeted
+	summary.NumTargeted = &temp_summary.NumTargeted
 	// NumRan is the number of hosts that actually ran the script successfully.
-	summary.NumRan = temp_summary.NumSucceeded
+	summary.NumRan = &temp_summary.NumSucceeded
 	// NumErrored is the number of hosts that errored out, which includes
 	// both failed and did not run.
-	summary.NumErrored = temp_summary.NumFailed + temp_summary.NumDidNotRun
+	summary.NumErrored = ptr.Uint(temp_summary.NumFailed + temp_summary.NumDidNotRun)
 	// NumFailed is the number of hosts that were canceled before execution.
-	summary.NumCanceled = temp_summary.NumCancelled
+	summary.NumCanceled = &temp_summary.NumCancelled
 	// NumPending is the number of hosts that are pending execution.
-	summary.NumPending = temp_summary.NumTargeted - (temp_summary.NumSucceeded + temp_summary.NumFailed + temp_summary.NumDidNotRun + temp_summary.NumCancelled)
+	summary.NumPending = ptr.Uint(temp_summary.NumTargeted - (temp_summary.NumSucceeded + temp_summary.NumFailed + temp_summary.NumDidNotRun + temp_summary.NumCancelled))
```

Comment:
this is all making the existing `BatchExecuteSummary` code compatible with the new `BatchActivity` type which has pointers to ints instead of ints.  This code will go away soon.